README.md

```diff
@@ -38,15 +39,40 @@ python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 
 ## Currently supported
 
-Operations - Contraction Tensor
-Data Types - FP32 , FP64
+### Operation: Contraction tensor
+
+hiptensor supports a tensor contraction of the form $D = \alpha AB + \beta C$
+
+Supported data-type combinations are:
+
+| typeA       | typeB       | typeC       | typeCompute       | notes                              |
+|-------------|-------------|-------------|-------------------|------------------------------------|
+| bf16        | bf16        | bf16        | f32               |                                    |
+| __half      | __half      | __half      | f32               |                                    |
+| f32         | f32         | f32         | bf16              |                                    |
+| f32         | f32         | f32         | __half            |                                    |
+| f32         | f32         | f32         | f32               |                                    |
+| f64         | f64         | f64         | f32               | f64 is supported only on gfx90a +  |
```

Comment:
I would suggest changing "is supported only on" to "is only supported on".

Reply:
changed to "is only supported on".